newIDE/app/src/InstancesEditor/InstancesResizer.js

```diff
@@ -220,6 +223,7 @@ export default class InstancesResizer {
         ? (initialSelectionAABB.height() + flippedTotalDeltaY) /
           initialSelectionAABB.height()
         : flippedTotalDeltaY;
+    let scaleZ = Math.max(scaleX, scaleY);
```

Comment:
Why this?

Reply:
Ok I guess this is for the proportional aspect, but I'm worried this will apply even if `proportional` is not set.